client/src/pages/Error404/Error404.tsx

```diff
@@ -0,0 +1,86 @@
+import { useNavigate } from 'react-router-dom';
+import React from 'react';
+import Error from '../../../public/Illustration/Error.svg';
```

Comment:
Move this `Error.svg` inside `src` directory and put it in the same folder in`components` directory as the new component you will make(see next review comment)